src/frontend/components/SøknadsSteg/VelgBarn/Barnekort/FjernBarnKnapp.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import styled from 'styled-components/macro';
+
+import { Knapp } from 'nav-frontend-knapper';
+
+import { DeleteFilled } from '@navikt/ds-icons';
+
+import { useApp } from '../../../../context/AppContext';
+import SpråkTekst from '../../../Felleskomponenter/SpråkTekst/SpråkTekst';
+
+const StyledKnapp = styled(Knapp)`
+    margin-top: 1rem;
+    margin-bottom: -1rem;
```

Comment:
Hva skjer her? Ved første øyekast skulle jeg tro at disse to bare resetter hverandre på en måte 🤔

Reply:
Hvordan resetter de hverandre? margin-top legger på avstand fra checkboxen, margin-bottom minsker avstand til bunnen av kortet.

Reply:
Ah, skjønner!